packages/experimental/MenuButton/src/renderContextualMenu.tsx

```diff
@@ -0,0 +1,48 @@
+/* @jsxFrag React.Fragment */
+import React, { useState, useCallback } from 'react';
+import { ContextualMenu, ContextualMenuItem, SubmenuItem, Submenu, ContextualMenuProps } from '@fluentui-react-native/contextual-menu';
+import { MenuButtonItemProps } from './MenuButton.types'
+
+export const renderContextualMenu = (contextualMenu: ContextualMenuProps, menuItems: MenuButtonItemProps[]): JSX.Element => {
```

Comment:
I'm guessing you made this a separate file so you could use the fragment? 
I like the separation, feels cleaner =)